shell_integration/MacOSX/OwnCloudFinderSync/OwnCloudFinderSync.xcodeproj/project.pbxproj

```diff
@@ -476,6 +494,7 @@
 				MTL_ENABLE_DEBUG_INFO = NO;
 				OC_APPLICATION_NAME = ownCloud;
 				OC_APPLICATION_REV_DOMAIN = com.owncloud.desktopclient;
+				OC_OEM_SHARE_ICNS = "";
```

Comment:
This doesn't seem to be enough to build it directly in Xcode, the shell script fails.